api4/role.go

```diff
@@ -29,12 +29,28 @@ var notAllowedPermissions = []string{
 }
 
 func (api *API) InitRole() {
+	api.BaseRoutes.Roles.Handle("", api.ApiSessionRequiredTrustRequester(getAllRoles)).Methods("GET")
```

Comment:
Should I  use ApiSessionRequired instead of ApiSessionRequiredTrustRequester?

Reply:
Yes, `ApiSessionRequired` would be the right one for these APIs.  `ApiSessionRequiredTrustRequester` is for situations where a user need not be logged in.